src/tool_operate.c

```diff
@@ -2328,6 +2328,16 @@ static CURLcode transfer_per_config(struct GlobalConfig *global,
       else {
         result = FindWin32CACert(config, tls_backend_info->backend,
                                  "curl-ca-bundle.crt");
+#if defined(USE_WIN32_CRYPTO) && \
+    !defined(CURL_WINDOWS_OPENSSL_NEVER_AUTOMATIC_USE_WINDOWS_CA_STORE) && \
+    !defined(LIBCURL_WINDOWS_OPENSSL_NEVER_USE_WINDOWS_CA_STORE)
+        if(!config->cacert && !config->capath) {
+          /* user, and environement did not specify any ca file or path
+             and there is no "curl-ca-bundle.crt" file in standard path
+             so the only possible solution is using the windows ca store */
+          config->cacert = strdup(CURL_WINDOWS_CA_STORE);
```

Comment:
This is probably what breaks test 1400 on Windows as this suddenly sets an option that wasn't set before. I don't think it should be implied like this when nothing else is found/specified, I think it would benefit from having an explicit option instead!

Reply:
`--cacert windows-ca-store` is perhaps not good enough, but something in that vein?

Reply:
I'm not a fan of piggybacking on the CACERT option to pass fake filename "///WINDOWS_CA_STORE" because other backends may treat it as a file not found error. What do you think of CURLOPT_SSL_OPTIONS option value CURLSSLOPT_FALLBACK_WINDOWS_CA

Reply:
I like the idea of putting it into CURLOPT_SSL_OPTIONS, with the exception that I don't think it should be a fallback, I think it should be explicitly asking for that or fail if it can't be fulfilled - if the library in question knows about the bit of course. If it doesn't, it will of course fallback to use the specified CA cert file. Which very well might be exactly what a typical application would want.

I also came to think of that maybe we should name the option so that we could potentially also use it in a future for importing other native CA stores, like the one on macOS.

Reply:
I pushed a new version with  CURLSSLOPT_NATIVE_CA